ui/components/app/transaction-list/transaction-list.component.js

```diff
@@ -115,6 +115,11 @@ export default function TransactionList({
 
   const pendingLength = pendingTransactions.length;
 
+  useEffect(() => {
+    const el = document.querySelector('.app');
```

Comment:
I like this approach, but I think we should put it at the page level instead of the transaction-list. pages/asset/asset.js is probably a better location.

Reply:
Thanks for your advice, I will try.

Reply:
Can you give me some advice on why the test failed?

Reply:
With one failed e2e and not the other likely a flakey test, triggered a rerun

Reply:
Hmm seems to be a legit fail.

One thing you could do is use `yarn test:e2e:single --retries 0 --browser firefox --leaveRunning <path to failed test file>` to see what the browser sees when it fails.

If you are still having issues Monday I'll see if I can help!

Reply:
## Circleci test
Failed from testing is
```
$ /home/circleci/project/node_modules/.bin/mocha --no-timeouts /home/circleci/project/test/e2e/tests/send-edit.spec.js


  Editing Confirm Transaction
    1) goes back from confirm page to edit eth value, gas price and gas limit


  0 passing (6s)
  1 failing

  1) Editing Confirm Transaction
       goes back from confirm page to edit eth value, gas price and gas limit:

      AssertionError [ERR_ASSERTION]: Expected values to be strictly equal:

'12.2' !== '2.2'

      + expected - actual

      -12.2
      +2.2
      
      at /home/circleci/project/test/e2e/tests/send-edit.spec.js:71:16
      at processTicksAndRejections (internal/process/task_queues.js:95:5)
      at withFixtures (test/e2e/helpers.js:87:5)
      at Context.<anonymous> (test/e2e/tests/send-edit.spec.js:20:5)
```

This tests requires that ether value should be 2.2. But it is 12.2 in chrome browser.

## Firefox(89.0.2) - It works!
![firefox](https://user-images.githubusercontent.com/17763340/123577360-a4be2a80-d80e-11eb-816e-1e3ff938414c.gif)

## Chrome(91.0.4472.114) - Failed!
![chrome](https://user-images.githubusercontent.com/17763340/123577369-a982de80-d80e-11eb-911c-5262e2bce523.gif)

## Chrome - Passed!
I just add one line in the test code(send-edit.spec.js). and it works. but I am not sure if this is the right solution.

```
44 ++  await driver.fill('.unit-input__input', '');
45 await driver.fill('.unit-input__input', '2.2');
```
![fix](https://user-images.githubusercontent.com/17763340/123577487-e4851200-d80e-11eb-99f7-04428b0fcd33.gif)